packs/minecraft_java_edition/README.md

```diff
@@ -0,0 +1,139 @@
+# Nomad Pack: Minecraft (Java Edition)
+
+![Nomad Pack: Minecraft (Java Edition)](https://assets.workloads.io/nomad-pack-registry/minecraft_java_edition.png)
+
+## Table of Contents
+
+<!-- TOC -->
+* [Nomad Pack: Minecraft (Java Edition)](#nomad-pack-minecraft-java-edition)
+  * [Table of Contents](#table-of-contents)
+  * [Requirements](#requirements)
+  * [Usage](#usage)
+    * [Inputs](#inputs)
+    * [Outputs](#outputs)
+  * [Author Information](#author-information)
+  * [License](#license)
+<!-- TOC -->
+
+## Requirements
+
+- HashiCorp Nomad `1.5.x` or newer
+- HashiCorp nomad-pack `0.0.1` or newer
+- Nomad Task Driver(s) for either [`docker`](https://developer.hashicorp.com/nomad/docs/drivers/docker) or [`podman`](https://developer.hashicorp.com/nomad/plugins/drivers/podman)
+
+## Usage
+
+The `minecraft_java_edition` Pack can be run with a local fileset (commonly used when developing the Pack), or via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry).
+
+A Pack available _locally_ may be run like so:
+
+```shell
+nomad-pack run ./packs/minecraft_java_edition
+```
+
+A Pack available via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry) may be run like so:
+
+```shell
+# add the Pack Registry
+nomad-pack registry add workloads github.com/workloads/nomad-pack-registry
+
+# run the Pack
+nomad-pack run minecraft_java_edition --registry=workloads
+```
+
+> **Note**
+>
+> For a more detailed description on how to use Nomad Pack, see [this guide](https://developer.hashicorp.com/nomad/tutorials/nomad-pack/nomad-pack-intro#basic-use).
+
```

Comment:
```suggestion
## Persistent Storage

Warning: By default this Pack deploys an ephemeral Minecraft server. To enable persistent storage, use the `volumes` [variable](./variables.hcl).

```

Reply:
So, on one side I like this - because it makes the Pack more universal,

On the other side, I don't like it, because overriding the `volumes` variable can be tricky. Let me ponder this!

Reply:
I've resolved this by adding a note that these volumes are required and how to configure them